<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameOverWidget</class>
 <widget class="QWidget" name="GameOverWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Game Over</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier</family>
    </font>
   </property>
   <property name="text">
    <string>Words Correct:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier</family>
    </font>
   </property>
   <property name="text">
    <string>Errors:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_correct">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier</family>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_errors">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier</family>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier</family>
    </font>
   </property>
   <property name="text">
    <string>/ total</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_again">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Again!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
